### PR TITLE
DOCS-2277 Modifying RUM Data, Context & Data Collected, and Unified Service Tagging Edits 

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds RUM services documentation introducing Unified Service Tagging.

### Motivation

DOCS-2277

### Additional Notes

Related PRs: [Browser](https://github.com/DataDog/browser-sdk/pull/1572), [Android](https://github.com/DataDog/dd-sdk-android/pull/937), and [iOS](https://github.com/DataDog/dd-sdk-ios/pull/868).

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/browser/modifying_data_and_context.md [hunk 1]
```diff
@@ -31,66 +33,76 @@ There are various ways you can modify the [data collected][1] by RUM, to support
...
         trackViewsManually: true,
         ...
     });
-```
-{{% /tab %}}
-{{< /tabs >}}
+    ```
+    {{% /tab %}}
+    {{% tab "CDN async" %}}
+    ```javascript
```
Comment (maycmlee) [inline content/en/real_user_monitoring/browser/modifying_data_and_context.md:56]: This line is showing up in the doc: https://a.cl.ly/bLuBrd6Y

**Review by maycmlee — COMMENTED** [review]
Looks great! I noticed a couple of code snippet annotations that are showing up in the doc.

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 2]
```diff
@@ -413,7 +413,7 @@
               tag: "Blog"
               text: "Monitor Mobile Vitals in Datadog"
     - action: pull-and-push-file
-      branch: master
+      branch: alai97/android-rum-services-doc-update
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:416]:
```suggestion
      branch: master
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 3]
```diff
@@ -458,14 +458,14 @@
   - repo_name: browser-sdk
     contents:
     - action: pull-and-push-file
-      branch: main
+      branch: alai97/rum-services-documentation-updates
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:461]:
```suggestion
      branch: main
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 4]
```diff
@@ -458,14 +458,14 @@
...
       globs:
       - 'packages/rum/README.md'
       options:
         dest_path: '/real_user_monitoring/browser/'
         file_name: '_index.md'
         front_matters:
-          dependencies: ["https://github.com/DataDog/browser-sdk/blob/main/packages/rum/README.md"]
+          dependencies: ["https://github.com/DataDog/browser-sdk/blob/alai97/rum-services-documentation-updates/packages/rum/README.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:468]:
```suggestion
          dependencies: ["https://github.com/DataDog/browser-sdk/blob/main/packages/rum/README.md"]
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 5]
```diff
@@ -613,14 +613,14 @@
               text: "Explore your services, resources, and traces"
 
     - action: pull-and-push-folder
-      branch: master
+      branch: alai97/ios-rum-services-doc-update
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:616]:
```suggestion
      branch: master
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 6]
```diff
@@ -613,14 +613,14 @@
...
       globs:
         - docs/rum_collection/*
       options:
         dest_dir: '/real_user_monitoring/ios/'
         path_to_remove: 'docs/rum_collection/'
         front_matters:
-          dependencies: [ "https://github.com/DataDog/dd-sdk-ios/blob/master/docs/rum_collection/" ]
+          dependencies: [ "https://github.com/DataDog/dd-sdk-ios/blob/alai97/ios-rum-services-doc-update/docs/rum_collection/" ]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:623]:
```suggestion
          dependencies: [ "https://github.com/DataDog/dd-sdk-ios/blob/master/docs/rum_collection/" ]
```

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 7]
```diff
@@ -314,6 +317,17 @@ If you're using [connected logs and traces][1], enable automatic logs injection
 [1]: /tracing/connect_logs_and_traces/
 {{% /tab %}}
 
+{{% tab "RUM & Session Replay" %}}
+
+If you're using [connected RUM and traces][1], specify the browser application in the `service` field, define the environment in the `env` field, and list the versions in the `version` field of your initialization file.
```
Comment (alai97) [inline content/en/getting_started/tagging/unified_service_tagging.md:322]: browser and/or mobile apps?

**Pull request merged**